python/environment.yml

```diff
@@ -70,7 +70,7 @@ dependencies:
  - pydap
  - pyepsg
  - pygeos
- - pystac-client==0.3.5  # https://github.com/stac-utils/pystac-client/issues/237
+ - pystac-client>=0.3.5  # https://github.com/stac-utils/pystac-client/issues/237
```

Comment:
This will need to be `==3.5` until the next release (0.5.0 I think), which will include https://github.com/stac-utils/pystac-client/issues/237.